{"name": "Получение заказа пользователя без авторизации", "status": "passed", "start": 1709681186966, "stop": 1709681187126, "uuid": "28883b29-60e9-4b7a-9a25-f56aa6af0c82", "historyId": "4789c7aac39e50078c2e0c4aa7b44d12", "testCaseId": "4789c7aac39e50078c2e0c4aa7b44d12", "fullName": "tests.test_get_order.TestApiGetUserOrders#test_no_auth_user_get_user_order_error", "labels": [{"name": "feature", "value": "Проверяем получение заказа пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_order"}, {"name": "subSuite", "value": "TestApiGetUserOrders"}, {"name": "host", "value": "DESKTOP-AQ71LTK"}, {"name": "thread", "value": "13036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_order"}]}